package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "plibrary",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
You don't need to commit `package-lock.json` file, you can remove this one.

Reply:
Alright how do i remove it?
Sorry I'm new to this...

Reply:
It's okay don't worry about, We can only learn things by doing it.

You have done a great job!